base/range.jl

```diff
@@ -415,6 +415,36 @@ struct OneTo{T<:Integer} <: AbstractUnitRange{T}
 end
 OneTo(stop::T) where {T<:Integer} = OneTo{T}(stop)
 OneTo(r::AbstractRange{T}) where {T<:Integer} = OneTo{T}(r)
+
+"""
+    oneto(n)
+
+Create an `AbstractRange` that behaves like to `1:n`. The returned
+range may be more efficient than using `1:n` since the lower limit
+is guaranteed to be one by the type system. The definition in `Base`
```

Comment:
```suggestion
is guaranteed to be 1 by the type system. The definition in `Base`
```
Otherwise this may be understood as "one" referring to "The definition". At least this is how I understood it at first reading.